mods/ts/weapons/superweapons.yaml

```diff
@@ -56,19 +56,19 @@ SuicideBomb:
 	Warhead@2Res: DestroyResource
 
 IonCannon:
-	ValidTargets: Ground, Air
+	ValidTargets: Ground, Water, Air
```

Comment:
I'd keep it in incase a TS mapmodder introduces navies, doing something akin to Insert_Namer's TD navy map series.